gradle-metric-schema/src/main/java/com/palantir/metric/schema/gradle/MetricSchemaMarkdownPlugin.java

```diff
@@ -32,33 +34,38 @@ public void apply(Project project) {
         project.getPluginManager().apply(MetricSchemaPlugin.class);
 
         TaskProvider<CreateMetricsManifestTask> createMetricsManifest =
-                project.getTasks().named(MetricSchemaPlugin.CREATE_METRICS_MANIFEST, CreateMetricsManifestTask.class);
+                project.getTasks().named(CreateMetricsManifestTask.NAME, CreateMetricsManifestTask.class);
 
-        TaskProvider<CheckMetricMarkdownTask> checkMetricsMarkdown = project.getTasks()
-                .register(MetricSchemaPlugin.CHECK_METRICS_MARKDOWN, CheckMetricMarkdownTask.class, task -> {
-                    task.getManifestFile().set(createMetricsManifest.flatMap(CreateMetricsManifestTask::getOutputFile));
-                    task.setMarkdownFile(project.file("metrics.md"));
-                    task.dependsOn(createMetricsManifest);
```

Comment:
This dependency is declared implicitly with the `getManifestFile` property, so this isn't necessary.